terraform/lib/dependabot/terraform/file_selector.rb

```diff
@@ -14,4 +14,12 @@ def terragrunt_files
   def terragrunt_file?(file_name)
     file_name != ".terraform.lock.hcl" && file_name.end_with?(".hcl")
```

Comment:
Maybe we can reuse the new `lock_file?` method here now:

```suggestion
    !lock_file?(file_name) && file_name.end_with?(".hcl")
```